archetypes/Pools/index.tsx

```diff
@@ -102,22 +129,70 @@ export const Browse: React.FC = () => {
 
     return (
         <>
-            <div className="container mt-0 md:mt-20">
-                <div className="p-4 md:pt-16 md:pb-12 md:px-8 lg:px-16 mb-4 shadow-xl rounded sm:rounded-2xl md:rounded-3xl bg-theme-background">
-                    <section className="mb-8">
-                        <h1 className="font-bold text-3xl mb-2 text-theme-text">Pools</h1>
-                        <p className="mb-1 text-cool-gray-500 dark:text-cool-gray-300 matrix:text-theme-text-secondary">
-                            Browse the available Tracer Pools and Pool Tokens.
-                        </p>
-                        <FilterBar state={state} dispatch={dispatch} />
-                    </section>
-                    <PoolsTable
-                        rows={sortedFilteredTokens}
-                        deltaDenotion={state.deltaDenotion}
-                        onClickMintBurn={handleMintBurn}
-                        showNextRebalance={state.rebalanceFocus === RebalanceEnum.next}
-                    />
-                </div>
+            <div className="container mt-0 md:mt-5">
+                <section className="mb-8">
+                    <h1 className="font-semibold text-3xl text-theme-text">Pools</h1>
+                    <FilterBar state={state} dispatch={dispatch} />
+                </section>
+                {!sortedFilteredTokens.length ? <Loading className="w-10 mx-auto mt-10" /> : null}
+                {Object.keys(groupedSortedFilteredTokens).map((key, index) => {
+                    const dataRows = groupedSortedFilteredTokens[key as any] as BrowseTableRowData[];
+                    return (
+                        <div
+                            key={index}
+                            className="p-4 md:p-8 lg:px-16 mb-4 shadow-xl rounded sm:rounded-2xl md:rounded-3xl bg-theme-background"
+                        >
+                            <div className="w-full h-20 my-5 p-4 flex justify-between divide-x-4 divide-cool-gray-200 dark:divide-cool-gray-900 overflow-x-auto whitespace-nowrap rounded-xl bg-cool-gray-50 dark:bg-theme-background-secondary">
+                                <div className="flex pr-10">
+                                    <Logo
+                                        className="inline mr-3 my-auto"
+                                        size="lg"
+                                        ticker={key.split('/')[0] as LogoTicker}
+                                    />
```

Comment:
Might need to reduce the size of this logo down a little. Designs have it as 32px.

figma.com/file/Gh6WzO8gLZWDmTvRbw8ikm/Tracer-Style-Guide-%26-Components?node-id=5993%3A204602

Reply:
Updated to 32px